sdks/python/apache_beam/internal/dill_pickler.py

```diff
@@ -46,9 +46,16 @@
 
 settings = {'dill_byref': None}
 
-if sys.version_info >= (3, 10) and dill.__version__ == "0.3.1.1":
-  # Let's make dill 0.3.1.1 support Python 3.11.
 
+def get_normalized_path(filename):
+  """Retruns the normalized path."""
+  normalized_filename = filename
```

Comment:
It is worth possibly adding a comment here that this is intended to be overridden so it doesn't get "optimized" away.